README.md

```diff
@@ -8,6 +8,30 @@ The site is built automatically with every push thanks to a [GH Actions](https:/
 
 If you want to build it locally, follow these simple steps:
 
+If you use windows, please execute
```

Comment:
This should be in a seperate section, maybe called `dependencies` or something similar.

Reply:
Thank you for your review. I have finished the revision as you said. Do you have any comments